src/core/itextsharp.csproj

```diff
@@ -18,9 +18,9 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net40' Or '$(TargetFramework)' == 'net45' ">
     <PackageReference Include="BouncyCastle" Version="1.8.1" />
+    <PackageReference Include="System.Drawing.Common" Version="4.5.0-preview1-26216-02" />

```

Comment:
System.Drawing.Common should go under the netstandard2.0 ItemGroup as it is not compatible with net40 or net45.